docs/08-spot-instances.yaml

```diff
@@ -0,0 +1,23 @@
+# A simple example of ClusterConfig object:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+    name: martina-test-spotinst-mixed-8
+    region: eu-central-1
+
+nodeGroups:
+    - name: ng-8
+      instanceType: t2.small
```

Comment:
I think when `instancesDistribution` is set, `instanceType` should be unset, we could also allow a special value (`mixed`) that can be used for explicitness.